Sources/NIOExtras/QuiescingHelper.swift

```diff
@@ -216,6 +216,7 @@ private final class CollectAcceptedChannelsHandler: ChannelInboundHandler {
 ///     try fullyShutdownPromise.futureResult.wait()
 ///
 public final class ServerQuiescingHelper {
+    public struct UnusedQuiescingHelperError: Error {}
```

Comment:
We should document this to say something like:

```suggestion
    /// The `ServerQuiescingHelper` was never used to create a channel handler.
    public struct UnusedQuiescingHelperError: Error {}
```